server/src/test-integration/java/com/thoughtworks/go/server/service/PipelineConfigServiceIntegrationTest.java

```diff
@@ -194,6 +194,22 @@ public void shouldCreatePipelineConfigWhenPipelineGroupDoesNotExist() throws Git
         assertThat(configRepository.getCurrentRevision().getUsername(), is(user.getDisplayName()));
     }
 
+    @Test
+    public void shouldCreatePipelineConfigWhenCaseInsensitivePipelineGroupIsSpecified() {
```

Comment:
This test is useful but we need to add a test where the pipeline group is created and ensure it does a case insensitive match.